systemd/dev_handler.service

```diff
@@ -1,15 +1,15 @@
 [Unit]
 Description=Device handler process for Runtime
-BindsTo=executor.service net_handler.service
+BindsTo=executor.service net_handler.service network_switch.service
 After=shm_start.service
```

Comment:
We had to modify all of the `systemd` service files to reflect the new home directory of the default user on the Le Potato, which was `ubuntu`. The home directory is now `/home/ubuntu` instead of `/home/pi`, as it was on the Raspberry Pi's.